src/Raven.Server/Commercial/SetupManager.cs

```diff
@@ -1126,6 +1135,23 @@ public void Configure(IApplicationBuilder app)
             }
         }
 
+        private static async Task<bool> CanGoogleResolveDns(string serverUrl, CancellationToken token)
+        {
```

Comment:
We are missing here an indication that the issue is really DNS.
What if this is a busy port? Firewall, etc? 
Need to validate that the error is with DNS by making DNS query and seeing it fail